### PR TITLE
RCPS:183-188-desktop-backend-maintenance-Refactor-Offense

### PR DESCRIPTION
**Pull request by Joven2002**
This request includes all desktop backend maintenance for refactoring offense. The code on this request has refactored accordingly to the coding standards. All unnecessary code has removed. And also The functionalities are completely working and there are no bugs that causes by refactoring.

### CODE REVIEW

**Review by JustineDaveMagnaye — APPROVED** [review]

**Review by zhairanicolepacheco — APPROVED** [review]

File: src/main/java/com/rc/porms/appl/facade/user/impl/UserFacadeImpl.java [hunk 1]
```diff
@@ -6,6 +6,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.List;
```
Comment (esdi-00) [inline src/main/java/com/rc/porms/appl/facade/user/impl/UserFacadeImpl.java:9]: where did you use this LIst?

File: src/main/java/com/rc/porms/data/user/dao/UserDao.java [hunk 2]
```diff
@@ -3,6 +3,8 @@
 
 import com.rc.porms.appl.model.user.User;
 
+import java.util.List;
```
Comment (esdi-00) [inline src/main/java/com/rc/porms/data/user/dao/UserDao.java:6]: List?

File: src/main/java/com/rc/porms/data/utils/user/QueryConstant.java [hunk 3]
```diff
@@ -6,4 +6,6 @@ public class QueryConstant {
         /**user query***/
         public static final String GET_USERS_BY_USERNAME_STATEMENT = "SELECT * FROM LOGIN WHERE username=?";
 
+
```
Comment (esdi-00) [inline src/main/java/com/rc/porms/data/utils/user/QueryConstant.java:9]: empty lines?

File: src/main/java/com/rc/porms/appl/facade/user/UserFacade.java [hunk 4]
```diff
@@ -5,6 +5,8 @@
 package com.rc.porms.appl.facade.user;
 import com.rc.porms.appl.model.user.User;
 
+import java.util.List;
```
Comment (esdi-00) [inline src/main/java/com/rc/porms/appl/facade/user/UserFacade.java:8]: unused imports

**Review by esdi-00 — CHANGES_REQUESTED** [review]
see comments

**Review by esdi-00 — APPROVED** [review]

**Pull request merged**